package.json

```diff
@@ -23,11 +23,13 @@
   "scripts": {
     "tslint": "tslint --project tsconfig.json",
     "tsc": "tsc --project tsconfig.json",
-    "ngc": "ngc --project tsconfig.aot.json",
-    "demo-ios": "grunt build && cd demo && tns run ios",
-    "demo-android": "grunt build && cd demo && tns run android",
-    "demo-ng-ios": "grunt build && cd demo-ng && tns run ios",
-    "demo-ng-android": "grunt build && cd demo-ng && tns run android"
+    "demo-ios": "grunt build && cd demo && ns run ios",
+    "demo-android": "grunt build && cd demo && ns run android",
+    "demo-ng-ios": "grunt build && cd demo-ng && ns run ios",
+    "demo-ng-android": "grunt build && cd demo-ng && ns run android",
+    "postinstall": "ngcc",
+    "setup": "npm i && ts-patch install",
```

Comment:
If the `setup` script is supposed to be run before build please add it to the grunt config. Otherwise the published packages will not be a working one.